website/src/markdown/api/useLocation.md

```diff
@@ -0,0 +1,19 @@
+# useLocation
+
+Returns the location to any component.
+
+This API requires a hook-compatible version of React.
+
+```jsx
+import { useLocation } from "@reach/router"
+
+const AnalyticTracker = (props) => {
+  const location = useLocation();
+
+  useEffect(() => {
+    ga.send(['pageview', location.pathname]);
+  }, [])
+
+  return null
+)
+```
```

Comment:
I'd just use a hook, we don't need these weird components that don't render anything anymore :D

```js
import { useLocation } from "@reach/router"

const usePageViews = () => {
  const location = useLocation();
  useEffect(() => {
    ga.send(['pageview', location.pathname]);
  }, [location]) // don't forget the location here:
)
```